pkg/asset/installconfig/powervs/regions.go

```diff
@@ -30,7 +30,11 @@ func IsKnownRegion(region string) bool {
 }
 
 func knownZones(region string) []string {
-	return powervs.Regions[region].Zones
+	zones := []string{}
+	for z := range powervs.Regions[region].Zones {
+		zones = append(zones, z)
+	}
+	return zones
```

Comment:
Why do you need to make a copy of the slice?

Reply:
> Why do you need to make a copy of the slice?

[powervs.Regions[region].Zones is a map whose keys are zone names](https://github.com/openshift/installer/pull/9245/files#diff-928d79e9ab16a144fb9cd4d12cf31e27fc681a0d23134fb63e0b7eee53536bf3R33). We need to return a slice of the zone names. Is there a better way to return just the keys?

Reply:
Oh I see. For some reason I skipped the part that changed it from a slice to a map.

It's not _better_ but we could use a generic function that could be employed for other similar "extract keys" scenarios:
```go
func Keys[K comparable, V any](m map[K]V) []K {
    keys := make([]K, 0, len(m))
    for k := range m {
        keys = append(keys, k)
    }
    return keys
}
```
That would need to be an installer utility function since we need it in other parts, so out of the scope for this PR.

Reply:
Agreed, may be useful elsewhere too. I'll add this function in a central place and submit a PR as I get time.